### ISSUE TITLE
Ideas for further enhancing color contrast?

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#609]
I know we're using axe-core for color contrast. I noticed while setting up the rule testing page for this rule that It's not failing elements that I expect it to fail.

Currently, axe-core is only catching 2 of the 4 failing items [on this page](https://a11ycheckrules.wpenginepowered.com/insufficient-color-contrast/).

We may want to circle back to this and see how we can enhance what Axe is seeing.

**Comment by SteveJonesDev** [comment]
WAVE returns two color contrast issues as well.

<img width="868" alt="Screenshot 2024-05-05 at 11 53 57 PM" src="https://github.com/equalizedigital/accessibility-checker/assets/2895788/68770aa0-2fd7-487d-9f33-7ad9532efa59">

**Comment by SteveJonesDev** [comment]
Unless you scale the page down and evaluate with smaller font sizes.

<img width="604" alt="Screenshot 2024-05-05 at 11 58 50 PM" src="https://github.com/equalizedigital/accessibility-checker/assets/2895788/447b37a8-34b3-4d4a-b291-f25ef6ab74ea">

**Comment by SteveJonesDev** [comment]
@amberhinds, I think returning two issues is correct. The font has a computed value of 24px therefore it meets AA color contrast for large text.

<img width="1435" alt="Screenshot 2024-05-06 at 12 05 01 AM" src="https://github.com/equalizedigital/accessibility-checker/assets/2895788/0729afa6-0af8-413a-81e3-5045cfec9b70">

<img width="722" alt="Screenshot 2024-05-06 at 12 04 22 AM" src="https://github.com/equalizedigital/accessibility-checker/assets/2895788/c62bbc01-87ef-42fa-8add-6910231e326d">

This does bring up the need to evaluate different viewport widths (Desktop, Tablet, Mobile).

**Comment by SteveJonesDev** [comment]
Issue moved to our internal system.
